README.md

```diff
@@ -1,2 +1,26 @@
-# jsweekend
-Project for attendees of https://jsweekend.cz
+# JS Weekend
+
+> An intensive weekend React workshop with Kiwi.com developers
+
+## About
+
+The goal of this workshop is to show you some great tools and best practices for
+building React application. During two weekend days you'll learn how to fetch
+data with Relay Modern from GraphQL backend, enrich Javascript with static
+types due to Flow, write test, create re-usable React components and setup
+continuous deployment.
+
+The workshop is divided into four blocks plus one short introduction. Each block
+corresponds to a directory containing the desired source code we'll try to create.
+
+## Installation
+
+There are no complicated dependencies. All you need is [Node.js](https://nodejs.org/en/)
+and [Yarn](https://yarnpkg.com/en/) installed, everything else will be installed
+from NPM.
```

Comment:
There is currently Node.js version restriction. It needs to be mentioned here unless someone will fix it.

It would be also great to have copy-paste commands to execute. Or it's not necessary to run Yarn here? Something like:

```